util/src/main/java/tech/pegasys/teku/util/config/TekuConfiguration.java

```diff
@@ -456,17 +457,30 @@ public void validateConfig() throws IllegalArgumentException {
               "Invalid configuration. Interop number of validators [%d] must be greater than or equal to [%d]",
               interopNumberOfValidators, Constants.SLOTS_PER_EPOCH));
     }
-    validateKeyStoreFilesAndPasswordFilesSize();
+    validateKeyStoreFilesAndPasswordFilesConfig();
   }
 
-  private void validateKeyStoreFilesAndPasswordFilesSize() {
+  private void validateKeyStoreFilesAndPasswordFilesConfig() {
     final List<String> validatorKeystoreFiles = getValidatorKeystoreFiles();
     final List<String> validatorKeystorePasswordFiles = getValidatorKeystorePasswordFiles();
 
+    if ((validatorKeystoreFiles != null && validatorKeystorePasswordFiles == null)
+        || (validatorKeystoreFiles == null && validatorKeystorePasswordFiles != null)) {
+      final String errorMessage =
+          "Invalid configuration. '--validators-key-files' and '--validators-key-password-files' cannot be specified separately";
+      throw new InvalidConfigurationException(errorMessage);
+    }
+
     if (validatorKeystoreFiles.size() != validatorKeystorePasswordFiles.size()) {
+      StatusLogger.getLogger()
+          .debug(
+              "Invalid configuration. The size of validator.validatorsKeystoreFiles {} and validator.validatorsKeystorePasswordFiles {} must match",
+              validatorKeystoreFiles.size(),
+              validatorKeystorePasswordFiles.size());
+
       final String errorMessage =
           String.format(
-              "Invalid configuration. The size of validator.validatorsKeystoreFiles [%d] and validator.validatorsKeystorePasswordFiles [%d] must match",
+              "Invalid configuration. The number of arguments for --validators-key-files [%d] and --validators-key-password-files [%d] must match",
```

Comment:
number of arguments doesn't seem quite right. The argument is a list, containing a number of items? Size or length might be more accurate

Reply:
We're trying to make it more human-readable, imo referring to the number of items in a list as size or length is confusing.

Perhaps something more like: 
`"The number of files specified for --validators-key-files [%d] and --validators-key-password-files [%d] must match"` ?

Reply:
`each key file specified by --validators-keys-files must also have a password file specified by --validators-key-password-files`?

Reply:
you must specify the same number of key and password files?

Reply:
The number of --validators-key-files (num) must equal the number of --validators-key-password-files (num)